ui/guest/src/react/I18nProvider.tsx

```
@@ -0,0 +1,24 @@
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { PropsWithChildren, useState } from 'react';
+import { IntlShape, RawIntlProvider } from 'react-intl';
+import { getCurrentIntl } from '../utils/i18n';
+
+export function I18nProvider(props: PropsWithChildren<{}>) {
+	const [intl] = useState<IntlShape>(getCurrentIntl());
+	return <RawIntlProvider children={props.children} value={intl} />;
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)